src/controllers/artist.js

```diff
@@ -0,0 +1,96 @@
+// src/controllers/artist.js
+const getDb = require('../services/db');
+
+exports.create = async (req, res) => {
+  const db = await getDb();
+  const { name, genre } = req.body;
+
+  try {
+    await db.query('INSERT INTO Artist (name, genre) VALUES (?, ?)', [
+      name,
+      genre,
+    ]);
+
+    res.status(201).send('Artist Created');
+  } catch (err) {
+    res.sendStatus(500).json(err);
+  }
+
```

Comment:
Could remove this blank line to keep the same format as the other functions in the file? No hard and fast rule for formatting but consistency is important